crates/moonbuild/src/upgrade.rs

```diff
@@ -289,10 +289,9 @@ pub fn do_upgrade(root: &'static str) -> Result<i32> {
 
                             // use new moon to bundle
                             let moon = moon_dir::home().join("bin").join("moon");
-                            println!("moon version:");
                             println!("Compiling {} ...", MOONBITLANG_CORE);
                             let out = std::process::Command::new(&moon).args(["version"]).output()?;
```

Comment:
we could have `moon version --all` here to show more info, and the following could be "moonbit toolchain version"